xtask/codegen/css.ungram

```diff
@@ -62,11 +62,19 @@ CssBogusKeyframesName = SyntaxElement*
 CssBogusUnicodeRangeValue = SyntaxElement*
 CssBogusSupportsCondition = SyntaxElement*
 
+AnyCssRoot =
+	CssRoot
+	| CssInlineRoot
```

Comment:
What if we call the new root `CssSnippetRoot` instead? It's in line with the new feature